app/controllers/posts_controller.rb

```diff
@@ -1,11 +1,13 @@
 class PostsController < ApplicationController
   def index
     @user = User.find_by(id: params[:user_id])
-    @posts = @user.posts
+    # @posts = @user.posts
+    @posts = @user.posts.includes(:comments)
   end
 
   def show
-    @post = Post.find_by(id: params[:id])
+    # @post = Post.find_by(id: params[:id])
```

Comment:
- [ ] Kindly remove all commented-out codes from this file. Removing commented-out code is good practice because it declutters the codebase, improves readability, reduces confusion, and ensures that only active and relevant code is maintained. 👍 💻

Reply:
Great work making this change and cleaning up your code.